app/models/commit_monitor_branch.rb

```diff
@@ -39,6 +39,14 @@ def pr_number
     MiqToolsServices::MiniGit.pr_number(name) if pull_request?
   end
 
+  def github_pr_uri
+    unless pull_request?
+      raise ArgumentError, "Cannot retrieve pull request URI for non-pull request " \
```

Comment:
WDYT about raising a different error here or returning nil instead? I'm not sure about using `ArgumentError` here since it's being raised here for reasons unrelated to this method's arguments.

Also, I see there is a class level `.github_commit_uri` method in this class. You could make this a class level method to make this more uniform, or convert the other to be an instance method.